code/game/area/LV624.dm

```diff
@@ -1,4 +1,15 @@
 //LV624 AREAS--------------------------------------//
+/obj/effect/landmark/lv624
```

Comment:
Merge garbage.

Reply:
That's actually intended (unless you can point me to a PR that does the same thing)

Reply:
Ah, unrelated coded cleanup, had me confused here.